documentation/develop.md

```diff
@@ -136,6 +136,9 @@ Also some options that may interest you:
  - `NO_CLEANUP=1` to not restart/clean state after test ends, used with `DEBUG=1`
  - `DEBUG=1` to keep the server running after the tests are done and to provide more detailed output
  - `MOCHA_WEBDRIVER_LOGDIR=/tmp/grist-tests` to specify the directory where the logs of the end-to-end tests will be stored (together with screenshots of the browser at the time of failure)
+ - `TYPEORM_DATABASE=/path/to/test-database.sqlite` (adapt the path to wherever you want the file to be created) in order to
+   debug tests implying a database. You may then inspect the database using the `sqlite3` command.
```

Comment:
"implying" is not the word here, maybe "that work with" or "that use"?

Reply:
Oh, I meant `involving`. But `that work with` also seem simpler and accurate.